documentation/IDTA-01001/modules/ROOT/pages/includes/IDTA-01001_ValueOnly.adoc

```diff
@@ -186,7 +214,20 @@ For a _SubmodelElementList,_ the struct is serialized as array denoted by square
 }
 ----
 
-For a _MultiLanguageProperty_ named “Label”, the payload is minimized to the following:
+If the _SubmodelElementList_ itself is the subject for ValueOnly the serialization is minimized to the following:
+
+[source,json,linenums]
+----
+[
+    "Martha ExampleFamily",
+    "Jonathan ExampleFamily",
+    "Clark ExampleFamily"
+]
+----
```

Comment:
This seems not to be correct since e.g. collections within a list do not have idShorts but this would be requested here.